packages/nouns-contracts/tasks/deploy-local-dao-v3.ts

```diff
@@ -50,7 +50,13 @@ task('deploy-local-dao-v3', 'Deploy contracts to hardhat')
   .addOptionalParam('auctionDuration', 'The auction duration (seconds)', 60 * 2, types.int) // Default: 2 minutes
   .addOptionalParam('timelockDelay', 'The timelock delay (seconds)', 60 * 60 * 24 * 2, types.int) // Default: 2 days
   .addOptionalParam('votingPeriod', 'The voting period (blocks)', 4 * 60 * 24 * 3, types.int) // Default: 3 days
-  .addOptionalParam('votingDelay', 'The voting delay (blocks)', 1, types.int) // Default: 1 block
+  .addOptionalParam('votingDelay', 'The voting delay (blocks)', 100, types.int) // Default: 1 block
+  .addOptionalParam(
+    'proposalUpdatablePeriodInBlocks',
+    'The updatable period in blocks',
+    100,
+    types.int,
+  ) // Default: 1 block
```

Comment:
inline comment says 1 block but it's now 100 blocks.
also thinking we should make this short again, maybe 10 blocks, wdyt?

Reply:
i needed a longer window for testing the updatable state. but it's easy to switch back.

Reply:
resolved here: https://github.com/nounsDAO/nouns-monorepo/pull/745/commits/2419f27ace2558034ed2291b05cc597f04dab2e2